<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Доходы/Расходы</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFDFDC;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>621</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #DB6B88;</string>
            </property>
            <property name="text">
             <string>Расходы</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="searc_r">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #DCABAE;
color: white;
font-weight: bold;
padding-left: 10px;
padding-right: 10px;
padding-bottom: 5px;
padding-top: 2px;
border: 1px solid #F5B2AC;
border-radius: 2px;</string>
              </property>
              <property name="text">
               <string>поиск</string>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="day_r">
                <property name="styleSheet">
                 <string notr="true">background-color: white;</string>
                </property>
                <property name="text">
                 <string>день</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="month_r">
                <property name="styleSheet">
                 <string notr="true">background-color: white;</string>
                </property>
                <property name="text">
                 <string>месяц</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="year_r">
                <property name="styleSheet">
                 <string notr="true">background-color: white;</string>
                </property>
                <property name="text">
                 <string>год</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="tableViewSpend">
            <property name="styleSheet">
             <string notr="true">color: black;
background-color: white;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #DB6B88;</string>
              </property>
              <property name="text">
               <string>Сумма:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sum_r">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: white;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="print_r">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #FF9CA1;
color: white;
border:1 px solid #E85668;
border-radius: 5px;
font-weight: bold;
padding-top: 5 px;
padding-bottom: 5px;
margin-top: 20 px;
margin-bottom: 10px;</string>
              </property>
              <property name="text">
               <string>Печать расходов</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #DB6B88;</string>
         </property>
         <property name="text">
          <string>Итого</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sum">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: white;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #DB6B88;</string>
            </property>
            <property name="text">
             <string>Доходы</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="searc_d">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #DCABAE;
color: white;
font-weight: bold;
padding-left: 10px;
padding-right: 10px;
padding-bottom: 5px;
padding-top: 2px;
border: 1px solid #F5B2AC;
border-radius: 2px;</string>
              </property>
              <property name="text">
               <string>поиск</string>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="day_d">
              <property name="styleSheet">
               <string notr="true">background-color: white;</string>
              </property>
              <property name="text">
               <string>день</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="month_d">
              <property name="styleSheet">
               <string notr="true">background-color: white;</string>
              </property>
              <property name="text">
               <string>месяц</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="year_d">
              <property name="styleSheet">
               <string notr="true">background-color: white;</string>
              </property>
              <property name="text">
               <string>год</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="tableViewIncoming">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: white;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #DB6B88;</string>
              </property>
              <property name="text">
               <string>Сумма:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sum_d">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: white;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="print_d">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #FF9CA1;
color: white;
border:1 px solid #E85668;
border-radius: 5px;
font-weight: bold;
padding-top: 5 px;
padding-bottom: 5px;
margin-top: 20 px;
margin-bottom: 10px;</string>
              </property>
              <property name="text">
               <string>Печать доходов</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="dohod"/>
    <addaction name="rashod"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="dohod">
   <property name="text">
    <string>Доходы</string>
   </property>
  </action>
  <action name="rashod">
   <property name="text">
    <string>Расходы</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
